gologr-wlog/gologr_test.go

```diff
@@ -0,0 +1,77 @@
+package gologrwlog
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/palantir/witchcraft-go-logging/wlog"
+	"github.com/palantir/witchcraft-go-logging/wlog/svclog/svc1log"
+	"github.com/stretchr/testify/assert"
+
+	// Use zap as logger implementation
+	_ "github.com/palantir/witchcraft-go-logging/wlog-zap"
+)
+
+type logEntry struct {
```

Comment:
This is fragile as the API might break in case the `wlog-zap` logEntry implementation changes.
Can you implement a mock logger that sends logs to a channel and assert on these messages?

Reply:
Instead of using a struct I am now using a JSONMatcher

following the internal tests: https://github.com/etherandrius/witchcraft-go-logging/blob/develop/wlog/svclog/svc1log/svc1logtests/tests.go#L106-L125

In terms of fragility it's the same, but I think it's acceptable